code/modules/reagents/chemistry_reagents/alcohol.dm

```diff
@@ -167,7 +167,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	name = "Gin"
 	description = "It's gin. In space. I say, good sir."
 	color = "#664300" // rgb: 102, 67, 0
-	taste_description = "an alcoholic christmas tree"
+	taste_description = "wood"
```

Comment:
Just googled up, and it says gin has a pine aroma... wood is too "beavery", is it really necessary?

Reply:
Make a suggestion and i'll merge it, this was just the simplest I could think up

Reply:
"pine" would suffice I guess.

Reply:
Feel free to suggest it

Reply:
```suggestion
	taste_description = "pine"
```
Miscommunication at its best.